occlum/run_occlum_passed_tests.sh

```diff
@@ -24,16 +38,33 @@ get_blocklist_subtests(){
 run_one_test(){
 	ret=""
 	if [ -f $TEST_BIN_DIR/$1 ]; then
-		get_blocklist_subtests $1
-		occlum exec /bin/$1 --gtest_filter=-$FILTER
+		printf "$GREEN running $1$NC \n"
+		if [[ $OPERATION_MODE = "ngo" ]];then
+			get_blocklist_subtests $1
+			get_ngo_blocklist_subtests $1
+			FILTER=$FILTER$NGO_FILTER
+			occlum exec /bin/$1 --gtest_filter=-$FILTER
+		else
+			get_blocklist_subtests $1
+			occlum exec /bin/$1 --gtest_filter=-$FILTER
+		fi
 		ret=$?
+		FILTER=""
 	else
 		echo "Warning: test does not exit"
 		ret=1
 	fi
 	return $ret
 }
 
+if [ -z $OPERATION_MODE ];then
+	printf "$GREEN Running test cases passed in occlum $NC\n"
+elif [ $OPERATION_MODE != "ngo" ];then
+	echo The parameter is wrong, you can use '"./run_occlum_passed_tests.sh ngo"' to run the tests passed in ngo
```

Comment:
It is better to quote the msg for `echo`

Reply:
Modification has been completed

Reply:
I mean line 63.

Reply:
Modification has been completed